### ISSUE TITLE
Cannot select external subtitles on default player

### ISSUE THREAD

**Issue by Me0wzz** [jarnedemeulemeester/findroid#287]
**Describe the bug**

On the subtitle selection menu in Player (Default Player), External subtitles are detected on the menu.

 But, external subtitl items are disabled, and I cannot select it (radio button). They are greyed out.

I didn't edit any settings. I just installed the app, logged in my Jellyfin instance and played a video.

**To Reproduce**
1. Play a video
2. Click subtitle selection menu
3. See subtitles list and make sure that external subtitles are visible on the menu.
5. External subtitles are not selectable / are disabled

**Expected behavior**
External subtitles menu item (radio button) should not be greyed out.
They should be selected flawlessly.

**Device info (please complete the following information):**
 - Device: Google Pixel 6
 - Android version: 13
 - App version: v0.10.1
 - Jellyfin version: 10.8.9

**Additional context**

Directory Structure

``````
TV Show 

    └── Season 01

             └── Episode S01EP1.mkv

             └── Episode S01EP1.kor.srt

             └── Episode S01EP1.smi

             └── Episode S01EP2.mkv

             └── Episode S01EP2.kor.srt

             └── Episode S01EP2.smi

             └── etc..

``````

**Comment by jarnedemeulemeester** [comment]
I have never seen `.smi` files. After a quick Google search those seem to be subtitle files.
Is it only the `.smi` files that don't work or are the other `.srt` files also not unavailable?

I also don't know the format of `.smi` files, are they a specific language? Because they don't have a language in their name.

**Comment by Me0wzz** [comment]
Both smi, srt cannot be loaded.
I changed both of them to kor.smi and kor.srt.
But, nothing changed.

**Comment by Me0wzz** [comment]
I noticed that if Findroid cannot select certain subtitles, Jellyfin official android client's Exoplayer also does not load subtitle properly.

For other tvshows/movies, Findroid's default player and Jellyfin's Exoplayer loads subtitles flawlessly.

I think this is not Findroid-related issue.

When playing videos that have subtitles selection problem with `mpv player option`, the player loads subtitles flawlessly.

See: https://github.com/jellyfin/jellyfin-android/issues/944

**Comment by mskpluk** [comment]
Not resolved in 2.5.0-beta2
